beetsplug/discogs.py

```diff
@@ -270,12 +270,17 @@ def get_album_info(self, result):
         # `autotag.apply_metadata`, and set `medium_total`.
         for track in tracks:
             track.media = media
-            track.medium_total = mediums
+            track.medium_total = mediums.count(track.medium)
```

Comment:
This looks right! It seems like [collections.Counter](https://docs.python.org/2/library/collections.html#collections.Counter) could be a good data structure too, but this is a straightforward solution as well.